assets/styles/settings/_fonts.scss

```
@@ -1 +1,48 @@
-// Put fonts here, please.
+@font-face {
+	font-family: 'Publico Headline';
```

```suggestion
    font-family: 'Publico Headline';
```

Tähän tiedostoon eksyny tabeja